### PR TITLE
Crud UI

### PR DESCRIPTION
**Pull request by cindykandie**
## On this branch:
- I created the 3 pages of the site: Feed, Post, My Post
- I also added a navigation and used React Router DOM to navigate the app
- I added details of the app in the Readme file for documentation of what the app does
- I styled the whole main parts of the app

## Future additions
- I will connect the entire app to the API and fetch using React Query
- The post creation form will use the react hook form
- I will add and style the comment section

### CODE REVIEW

File: src/pages/Post.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+function Post() {
+  const [title, setTitle] = useState('');
+  const [body, setBody] = useState('');
+
+  // Form submission
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    // Send the title and body values to the api
+    console.log('Title:', title);
```
Comment (jgachunga) [inline src/pages/Post.jsx:11]: remove console logs

File: src/pages/MyPosts.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+
+function MyPosts() {
+  const [userPosts, setUserPosts] = useState([]);
+
+  // Simulated API call to fetch user's posts
+  useEffect(() => {
+    // API call to fetch user-specific posts
+    fetch('https://jsonplaceholder.typicode.com/posts?userId=1') // Replace 1 with the user's ID
```
Comment (jgachunga) [inline src/pages/MyPosts.jsx:9]: use axios, create a separate function to fetch then call the function in useEffect, dont call fetch directly in useEffect

**Review by jgachunga — CHANGES_REQUESTED** [review]
I did not see the different forms for adding a post,

**Comment by cindykandie** [comment]
> I did not see the different forms for adding a post,

The code for this is on the Post.jsx file that has one form to add title and body

**Review by jgachunga — APPROVED** [review]

**Pull request merged**